FirebaseAuth/Sources/Backend/RPC/FIRVerifyPasswordRequest.h

```diff
@@ -52,6 +52,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic, copy, nullable) NSString *captchaResponse;
```

Comment:
I'm curious, how come we already have this? Is it for previous recaptcha versions?

Reply:
Yeah, it's introduced long time ago but never used. The BE folks decide to reuse this field.